clowdapp.yaml

```diff
@@ -473,3 +525,6 @@ parameters:
 - name: IMPORT_SSG_SCHEDULE
   description: Cronjob schedule for import-ssg
   value: "*/5 * * * *" # every 5 minutes
+- name: MAX_INIT_TIMEOUT_SECONDS
+  description: Number of seconds for timeout init container operation
+  value: "120"
```

Comment:
:thinking: I would maybe rename this more explicitly to something related to maximum SSG import wait(ing?) time(?out?). Also 240 seconds sound more reasonable as the import takes like ~5 minutes in ephemeral. 

That's another thing that locally for me it takes only ~3 minutes and it is not even parallelized yet. With Marley's rule grouping I measured a 100-120 seconds longer import time.

Reply:
Well, we can discuss about names, but this is a timeout value for the init operation of whichever pod we're using - that is, the import-ssg and the run-migrations tasks - they're 2 different scripts, with very similar implementation. 

That said, the good thing is we can define this env variable per environment, so we can tune it based on which environment we're targeting. 

I can create 2 variables, for the 2 different init tasks timeout, and we can tune that per-environment on our deployment templates per namespace. LMK what you think